interfaces/DOM-Parsing.idl

```diff
@@ -0,0 +1,32 @@
+
+    [Constructor, Exposed=Window]
+    interface DOMParser {
+      [NewObject] Document parseFromString(DOMString str, SupportedType type);
+    };
+
+    enum SupportedType {
+      "text/html",
+      "text/xml",
+      "application/xml",
+      "application/xhtml+xml",
+      "image/svg+xml"
+    };
+
+    [Constructor, Exposed=Window]
+      interface XMLSerializer {
+      DOMString serializeToString(Node root);
+    };
+
+    partial interface Element {
+      [CEReactions, TreatNullAs=EmptyString] attribute DOMString innerHTML;
+      [CEReactions, TreatNullAs=EmptyString] attribute DOMString outerHTML;
+      [CEReactions] void insertAdjacentHTML(DOMString position, DOMString text);
+    };
+
+    partial interface Range {
+      [CEReactions, NewObject] DocumentFragment createContextualFragment(DOMString fragment);
+    };
+
+    partial interface Text {
+      attribute boolean serializeAsCDATA;
```

Comment:
I've added a test, and a failure pointed to something odd here. `serializeAsCDATA` isn't actually in the spec, it's commented out:
https://github.com/w3c/DOM-Parsing/blob/ba763b4d81a5a1fb5ec37deebf93659d853850e7/index.html#L608

@mdittmer any idea why that might be happening? Isn't a real HTML parser being used?

Reply:
Nope.